tests/tests.js

```diff
@@ -1,45 +1,177 @@
+const chai = require('chai')
+const sinon = require('sinon')
+const sinonChai = require('sinon-chai')
 const { expect } = require('chai')
```

Comment:
Since you already imported the full chai library you can extract expect off the existing object rather than reimporting chai again.  `const { expect } = chai`